client/my-sites/stats/activity-log-banner/error-banner.jsx

```diff
@@ -51,34 +50,28 @@ class ErrorBanner extends PureComponent {
 			rewindRestore,
 			createBackup,
 		} = this.props;
-		if ( requestedRestoreActivityId ) {
-			rewindRestore( siteId, requestedRestoreActivityId );
-		} else if ( downloadId ) {
+		if ( downloadId ) {
 			createBackup( siteId, downloadId );
+		} else if ( requestedRestoreActivityId ) {
+			rewindRestore( siteId, requestedRestoreActivityId );
```

Comment:
our more common pattern is early abort rather than `if/else`

```js
if ( downloadId ) {
  return createBackup( siteId, downloadId );
}

if ( requestedRestoreActivityId ) {
  return rewindRestore( siteId, requestedRestoreActivityId );
}
```

Reply:
Done, thanks.